src/util.cc

```diff
@@ -356,6 +360,23 @@ void SetMethod(Local<v8::Context> context,
   function->SetName(name_string);  // NODE_SET_METHOD() compatibility.
 }
 
+void SetMethod(v8::Isolate* isolate,
+               v8::Local<v8::Template> that,
+               const char* name,
+               v8::FunctionCallback callback) {
+  Local<v8::FunctionTemplate> t =
```

Comment:
I think we can now just reuse this from the variant of `SetMethod()` that takes a context?

Reply:
As `v8::Object::Set` and `v8::Template::Set` are distinct APIs and not compatible in their signatures, I didn't find those two methods can be reused in a way that is meaningful. Would you mind sharing a snippet of your ideas if possible? Thank you.

Reply:
I *think* if you set the name on the template V8 would just copy that over to the function when you create it from the template (and that might be slightly more efficient too?) If somehow we still have to set the name again, we can still reuse the template creation code and then just call set again on the function?

Reply:
Oh right this is a different template, so I guess we can't do that. Ignore this comment then.